components/mosart/src/riverroute/MOSART_physics_mod.F90

```diff
@@ -268,6 +325,81 @@ subroutine Euler
               THeat%Ha_eroutUp(iunit) = avdst_upstrm%rAttr(nt_rtm+1,cnt)
           end if
        enddo
+       
+       if (Tctl%RoutingMethod == 4 ) then
```

Comment:
What does `4` mean? Conder defining a suitable variable (e.g. `WAT_DTH_FRM_DWNSTR_CHANNEL`) and using it in the `if` block.

Reply:
4 means diffusion wave routing method. We will discuss this. Since it will affect a lot of code, let's leave it to the future.